### PR TITLE
Add Cypress and Playwright instructions to bun testing

### PR DESCRIPTION
**Pull request by Afrowave**
#### Description (required)
This PR adds instruction to the Bun runtime.

- Link to `bun test` instructions
- How to install Cypress 
- How to install Playwright

### CODE REVIEW

File: src/content/docs/en/recipes/bun.mdx [hunk 1]
```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
 bun ./dist/server/entry.mjs
 ```
 
+## Testing in Bun
+
+Bun ships with a fast, built-in, Jest-compatible test runner through the `bun test` command. If you like to use that read [`bun test` documentation](https://bun.sh/docs/cli/test).
+
+However, you may want to use Cypress or Playwright for a modern approach to testing web apps.
```
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:124]:
```suggestion
However, it is also possible to use Cypress or Playwright for a modern approach to testing web apps.
```
I feel like this slight adjustment introduces the sections below a little better. "You might want to use..." is kind of a flat, "so what?" statement on its own that feels like it needs a follow up, "If so, then..."  Just saying stuff is possible gets to the point a little quicker!

File: src/content/docs/en/recipes/bun.mdx [hunk 2]
```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
...
+
+### Cypress
+
+Cypress is a front-end testing tool and is on a mission to "make the testing experience enjoyable and generate developer happiness". This enables you to write end-to-end tests for your Astro site.
+
+You can install Cypress like so:
```
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:130]:
```suggestion
Install Cypress with the following command:
```
Whenever it makes reasonable sense, we prefer to give direct instructions. So, something like the above, or, "To install Cypress, use the following command:"

You have to imagine that the response to "you can..." is always, "I know I *can*... should I? Do I want to? Is there another way?"  Just tell people what to do!  It makes them feel more confident in what's going to happen!

File: src/content/docs/en/recipes/bun.mdx [hunk 3]
```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
...
+
+```bash
+bun add cypress --dev
+```
+
+For the rest of the configuration and to start your first test, follow the rest of Cypress process in the [Astro Testing Guide](https://docs.astro.build/en/guides/testing/#configuration).
```
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:136]: This is a beautiful sentence, and an excellent use of linking. No notes. Much respect.

File: src/content/docs/en/recipes/bun.mdx [hunk 4]
```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
...
+
+### Playwright
+
+Playwright is an end-to-end testing framework, which means that it allows you to test your Astro code on all modern rendering engines including Chromium, WebKit, and Firefox.
```
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:140]:
```suggestion
Playwright is an end-to-end testing framework that allows you to test your Astro code on all modern rendering engines including Chromium, WebKit, and Firefox.
```
We can just tighten this up a bit!

File: src/content/docs/en/recipes/bun.mdx [hunk 5]
```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
...
+
+You can install Playwright using the following command:
```
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:142]:
```suggestion
Install Playwright using the following command:
```

**Review by sarah11918 — COMMENTED** [review]
Hey @Afrowave ! Thank you so much for this contribution! I've made some suggestions for you just to conform a tiny bit more to our normal style/conventions, so see what you think! Happy to get this into docs!

**Comment by sarah11918** [comment]
Glad we're on the same page, @Afrowave ! Just a heads up that clicking "resolve" doesn't actually MAKE the changes. In fact, it signals the opposite to GitHub. "Nope, I am DONE with this comment!" lol  :smile: 

In future if (when, hopefully!) you contribute more, there should be a button for you to click that is "commit suggestion" and THAT means you've accepted the proposed change, and will update the file itself.

So, I'm going to go ahead manually and put my edits in there myself. In future, you can just "accept" suggestions that you approve of, and it will happen automatically.  :raised_hands:

File: src/content/docs/en/recipes/bun.mdx (hunk 1 above)
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:124]:
```suggestion
However, it is also possible to use Cypress or Playwright for a modern approach to testing web apps.
```

File: src/content/docs/en/recipes/bun.mdx (hunk 2 above)
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:130]:
```suggestion
Install Cypress with the following command:
```

File: src/content/docs/en/recipes/bun.mdx (hunk 4 above)
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:140]:
```suggestion
Playwright is an end-to-end testing framework that allows you to test your Astro code on all modern rendering engines including Chromium, WebKit, and Firefox.
```

File: src/content/docs/en/recipes/bun.mdx (hunk 5 above)
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:142]:
```suggestion
Install Playwright using the following command:
```

**Comment by sarah11918** [comment]
Oh, another issue... apparently I *can't* make these edits myself. In future, when you make a PR the box to "allow maintainers to make changes" will be checked by default. Please don't uncheck that (and, ensure that it *is* checked), because that's what allows me to make tiny changes directly so that we don't have to put the ball back in your corner for something very small.

Which means... the ball is back in your corner! You now need to "accept/commit" my proposed changes (not "resolve conversation") in order for those changes to actually be applied to your file. Thanks!

**Comment by Afrowave** [comment]
Thank you @sarah11918 for the very nice hand holding. Working on one's own repos never brings these kinds of interactions to bare. Wow! 😃

File: src/content/docs/en/recipes/bun.mdx [hunk 6]
```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
...
+
+Playwright is an end-to-end testing framework that allows you to test your Astro code on all modern rendering engines including Chromium, WebKit, and Firefox.
+
+Install Playwright using the following command:
+
+```bash
+bun create playwright
+```
+
+To create your first Playwright test, follow the intructions for the rest of the Playwright process in the [Astro Testing Guide](https://docs.astro.build/en/guides/testing/#create-your-first-playwright-test).
```
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:148]:
```suggestion
For the rest of the configuration and to start your first test, follow the rest of Cypress process in the [Astro Testing Guide](/en/guides/testing/#configuration).

### Playwright

Playwright is an end-to-end testing framework that allows you to test your Astro code on all modern rendering engines including Chromium, WebKit, and Firefox.

Install Playwright using the following command:

```bash
bun create playwright
```

To create your first Playwright test, follow the intructions for the rest of the Playwright process in the [Astro Testing Guide](/en/guides/testing/#create-your-first-playwright-test).
```

**Pull request merged**